runtime/doc/diff.txt

```diff
@@ -1,39 +1,31 @@
-*diff.txt*      For Vim version 7.4.  Last change: 2014 May 20
+*diff.txt*      For Vim version 7.4.  Last change: 2015 Jan 19
 
 
 		  VIM REFERENCE MANUAL    by Bram Moolenaar
 
 
-				*diff* *vimdiff* *gvimdiff* *diff-mode*
-This file describes the |+diff| feature: Showing differences between two,
-three or four versions of the same file.
+							*diff* *diff-mode*
+This file describes |diff-mode|, which shows the differences between two,
+three, or four versions of the same file.
 
 The basics are explained in section |08.7| of the user manual.
 
-1. Starting diff mode		|vimdiff|
+1. Starting diff mode		|starting-diff-mode|
```

Comment:
I didn't realize the old tag pointed right above this TOC entry, that's pretty silly. 

But, currently `|starting-diff-mode|` points to nowhere, it doesn't have a tag (a tag is defined with `*`). So just delete `starting-` and then it's fine.